### ISSUE TITLE
[Bug] Vampirism Infinity Book Replaces Vanilla Infinity Book?

### ISSUE THREAD

**Issue by Yourname942** [TeamLapen/Vampirism#662]
## Versions
- Minecraft: 12.2.2                   
- Forge: 14.23.5.2847                      
- Vampirism: Vampirism-1.12.2-1.6.2.jar                                            

## Issue Description
The vanilla Minecraft Infinity enchantment book is being replaced by Vampirism's Infinity book. This means that we can no longer use Infinity on regular vanilla bows nor regular bows from other mods.

## Reproduce Steps

Open Just Enough Items
Search for Infinity
Only Vamprism's Infinity book exists in game now (you can no longer acquire the regular infinity book)

**Comment by PendragonII** [comment]
The items share the same ID, `minecraft:enchantment_book` with different NBT tags and lore denotations. Going into creative mode inventory reveals that both Vanilla Minecraft and Vampirism infinity enchantment books coexist. Just as well, after multiple stacks of normal books used in the enchantment table I was able to obtain both Vanilla Minecraft and Vampirism infinity enchantment books.

Both types of infinity appear rarely within the enchantment table.

Tested on:
- Minecraft: 1.12.2
- Vampirism: 1.6.2
- Forge: 14.23.5.2844
